README.md

```diff
@@ -38,6 +38,13 @@ $ cd netplugin; make demo
 $ vagrant ssh netplugin-node1
 ```
 
+Optionally, variables can be passed to Makefile if needed. For example, to
+use 1GB memory for the vagrant VMs, run:
+
+```
+CONTIV_MEMORY=1024 make demo
```

Comment:
Can you change this to 3072 or 4096, please? Using VMs with 1 GB of RAM will most likely be something we won't support officially. The person who tries this would get to keep all the broken pieces if the VMs with 1 GB of RAM break the environment.

Reply:
Done, see latest commit. Do we need a note in the docs to let the user know that 1 GB of RAM must be avoided for the vagrant VMs as it breaks Contiv?